examples/ct/ct_server/main.go

```diff
@@ -19,9 +19,9 @@ import (
 
 // TODO(Martin2112): We still have the treeid / log ID thing to think about + security etc.
 var logIDFlag = flag.Int64("log_id", 1, "The log id (tree id) to send to the backend")
-var rpcBackendFlag = flag.String("log_rpc_backend", "localhost:8090", "Backend Log RPC server to use")
+var rpcBackendFlag = flag.String("log_rpc_server", "localhost:8090", "Backend Log RPC server to use")
 var rpcDeadlineFlag = flag.Duration("rpc_deadline", time.Second*10, "Deadline for backend RPC requests")
-var serverPortFlag = flag.Int("port", 8091, "Port to serve CT log requests on")
+var serverPortFlag = flag.Int("http_port", 6962, "Port to serve CT log requests on")
```

Comment:
The RPC servers have `--port` for RPC port and `--http_port` for HTTP port (just for metrics), so I thought `--http_port` was more consistent.  But the argument that `--port` is the main port that work comes in on is persuasive too.  Happy either way -- you call.